### PR TITLE
Trivial: Restrict source files in app/ledger to 80 columns.

### PR DESCRIPTION
**Pull request by rec**
@nbougalis @seelabs 

I have a script that does a bad job of this - but also directs me to each line in my editor so I can correct it by hand.

I'm going to do one of these every time I send out a more substantial review, just because these really long lines get me down.

### CODE REVIEW

File: src/ripple/websocket/AutoSocket.h [hunk 1]
```diff
@@ -207,61 +219,70 @@ class AutoSocket
     }
 
     template <typename Seq, typename Condition, typename Handler>
-    void async_read_until (const Seq& buffers, Condition condition, Handler handler)
+    void async_read_until(
```
Comment (seelabs) [inline src/ripple/websocket/AutoSocket.h]: space before paren

File: src/ripple/websocket/AutoSocket.h [hunk 2]
```diff
@@ -207,61 +219,70 @@ class AutoSocket
...
+        const Seq& buffers, Condition condition, Handler handler)
     {
-        if (isSecure ())
-            boost::asio::async_read_until (*mSocket, buffers, condition, handler);
+        if (isSecure())
+            boost::asio::async_read_until(
```
Comment (seelabs) [inline src/ripple/websocket/AutoSocket.h]: space before paren

File: src/ripple/websocket/AutoSocket.h [hunk 3]
```diff
@@ -207,61 +219,70 @@ class AutoSocket
...
+                *mSocket, buffers, condition, handler);
         else
-            boost::asio::async_read_until (PlainSocket (), buffers, condition, handler);
+            boost::asio::async_read_until(
```
Comment (seelabs) [inline src/ripple/websocket/AutoSocket.h]: space before paren

File: src/ripple/websocket/AutoSocket.h [hunk 4]
```diff
@@ -207,61 +219,70 @@ class AutoSocket
...
+                PlainSocket (), buffers, condition, handler);
     }
 
     template <typename Allocator, typename Handler>
-    void async_read_until (boost::asio::basic_streambuf<Allocator>& buffers, std::string const& delim, Handler handler)
+    void async_read_until(boost::asio::basic_streambuf<Allocator>& buffers,
```
Comment (seelabs) [inline src/ripple/websocket/AutoSocket.h]: We lost the space before the paren in many of these changes. I won't call each one out.

File: src/ripple/app/ledger/impl/InboundLedger.cpp [hunk 5]
```diff
@@ -466,8 +471,8 @@ void InboundLedger::trigger (Peer::ptr const& peer)
                 Message::pointer packet (std::make_shared <Message> (
                     tmBH, protocol::mtGET_OBJECTS));
                 {
-                    for (PeerSetMap::iterator it = mPeers.begin (), end = mPeers.end ();
-                            it != end; ++it)
+                    for (auto it = mPeers.begin (), end = mPeers .end ();
```
Comment (seelabs) [inline src/ripple/app/ledger/impl/InboundLedger.cpp]: No space after mPeers

File: src/ripple/app/ledger/impl/InboundLedgers.cpp [hunk 6]
```diff
@@ -99,7 +100,8 @@ class InboundLedgersImp
         {
             ScopedLockType sl (mLock);
 
-            hash_map<uint256, InboundLedger::pointer>::iterator it = mLedgers.find (hash);
+            auto it = mLedgers.
+            find (hash);
```
Comment (seelabs) [inline src/ripple/app/ledger/impl/InboundLedgers.cpp]: Make this one line

File: src/ripple/app/ledger/impl/LedgerCleaner.cpp [hunk 7]
```diff
@@ -301,7 +311,8 @@ class LedgerCleanerImp
 
         if(! getApp().getLedgerMaster().fixIndex(ledgerIndex, ledgerHash))
         {
-            m_journal.debug << "ledger " << ledgerIndex << " had wrong entry in history";
+            m_journal.debug << "ledger " << ledgerIndex
+                            << " had wrong entry in history";
```
Comment (seelabs) [inline src/ripple/app/ledger/impl/LedgerCleaner.cpp]: I think most ripple code puts the operator `<<` at the end of the previous line and indents the next line 4. Tho I don't think there's an official style guide for this.

File: src/ripple/app/ledger/impl/LedgerConsensusImp.cpp [hunk 8]
```diff
@@ -381,7 +388,8 @@ Json::Value LedgerConsensusImp::getJson (bool full)
             Json::Value ctj (Json::objectValue);
             for (auto& ct : mCloseTimes)
             {
-                ctj[beast::lexicalCastThrow <std::string> (ct.first)] = ct.second;
+                ctj[std::to_string(ct.first)] = ct.
+                second;
```
Comment (seelabs) [inline src/ripple/app/ledger/impl/LedgerConsensusImp.cpp]: I don't like splitting the member and the object. I'd put `ct.second` all on the second line.

File: src/ripple/websocket/Connection.h [hunk 9]
```diff
@@ -259,7 +259,7 @@ Json::Value ConnectionImpl <WebSocket>::invokeCommand (Json::Value& jvRequest)
     Json::Value jvResult (Json::objectValue);
 
     auto required = RPC::roleRequired (jvRequest[jss::command].asString());
-    Role const role = requestRole (required, m_port, jvRequest, m_remoteAddress);
+    auto role = requestRole (required, m_port, jvRequest, m_remoteAddress);
```
Comment (seelabs) [inline src/ripple/websocket/Connection.h]: Keep the const

**Comment by seelabs** [comment]
:+1: All of my comments were minor.

File: src/ripple/app/ledger/impl/LedgerCleaner.cpp (hunk 7 above)
Comment (vinniefalco) [inline src/ripple/app/ledger/impl/LedgerCleaner.cpp]: I could add it to the official style, what do you think?

Comment (HowardHinnant) [inline src/ripple/app/ledger/impl/LedgerCleaner.cpp]: I find the leading `<<` on the continuation line quite readable.  Though I don't know if we want to dictate style down to this degree.  I'm in favor of the official style promoting readable code, as opposed to where to place `<<` in a situation like this.

Comment (vinniefalco) [inline src/ripple/app/ledger/impl/LedgerCleaner.cpp]: @HowardHinnant Explain why `<<` at the beginning of a line is better than `<<` at the end of the previous line

Comment (HowardHinnant) [inline src/ripple/app/ledger/impl/LedgerCleaner.cpp]: It is subjective.  I am biased as a left-to-right reader.  My eyes tend to see the left of a line first, and so I like to put the most important information first.  To me it is very important to recognize that this is a continuation line, and what it is a continuation of.  I find lines that start with `<<`, where that `<<` is vertically lined up with a previous `<<` quite indicative of a multi-line streaming operation.

I do not feel strongly enough about this style to mandate it across the project.  I think having a somewhat permissive style guide is good in that it allows us to experiment with readability such as you have been doing with more frequent line breaks.  As long as the end result is readable (a very subjective term), I think we should allow it.

File: src/ripple/app/ledger/impl/InboundLedger.cpp (hunk 5 above)
Comment (nbougalis) [inline src/ripple/app/ledger/impl/InboundLedger.cpp]: I'd rather see a [FOLD] commit that changes this to:

    for (auto const& it : mPeers)

File: src/ripple/app/ledger/impl/InboundLedgers.cpp (hunk 6 above)
Comment (nbougalis) [inline src/ripple/app/ledger/impl/InboundLedgers.cpp]: Yeah, this is bad...

Comment (HowardHinnant) [inline src/ripple/app/ledger/impl/InboundLedgers.cpp]: I applaud the style experiment.  If experiments never failed, they wouldn't be experiments.  This one failed.  I agree, this fails my (admittedly subjective) "readability" test.

Comment (vinniefalco) [inline src/ripple/app/ledger/impl/InboundLedgers.cpp]: I didn't even realize it was a single statement

**Comment by rec** [comment]
We have stopped doing "space before paren" in big chunks of new code, so I've stopped doing it too!

File: src/ripple/websocket/AutoSocket.h (hunk 4 above)
Comment (rec) [inline src/ripple/websocket/AutoSocket.h]: As I said, that's our new style.

**Comment by rec** [comment]
Are we good to go?

**Comment by vinniefalco** [comment]
Can we squash this down?

**Comment by rec** [comment]
Squashed.

**Pull request closed without merging**